_layouts/posts.html

```diff
@@ -27,15 +27,28 @@
                     <a href="{{ post.url }}"><p>{{ post.title | escape }}</p></a>
                 {% endif %}
                 </div>
-
```

Comment:
The content from the line 9 to 52 could be moved into a file `_includes/post.html` and called here and in the `_layouts/categories.html` with `{% include _includes/post.html %}`. So it  makes it coherent between the page and removes the redundancy so easier to update/fix.
The same trick is used for people for example in `people.html` or several other location

Reply:
i left out 36 to 45 because the post are already categorized under the news/blog heading 
the post.excerpt was left out because of varying lenghth of the paragraphs 
I thought it would be a more basic card and less jampacked